packages/analytics-browser/src/utils/query-params.ts

```diff
@@ -1,6 +1,6 @@
 export const getQueryParams = (): Record<string, string | undefined> => {
   /* istanbul ignore if */
-  if (typeof window === 'undefined') {
+  if (typeof window?.location?.search === 'undefined') {
```

Comment:
I think accessing an `undefined` global variable throws an error. The `typeof` op above requires accessing `window`. Consider the following example:

```ts
if (typeof window?.location?.search) {
```

transpiles to:

```js
var _a;
if (typeof ((_a = window === null || window === void 0 ? void 0 : window.location) === null || _a === void 0 ? void 0 : _a.search)) {
```

It appears that the code accesses `window` object directly which will throw:

```
Uncaught ReferenceError: window is not defined
```

So I think it's probably best to check `typeof` window on its own.

```suggestion
  if (typeof window === 'undefined' || !window.location?.search) {
```

transpiles to:

```js
var _a;
if (typeof window === 'undefined' || !((_a = window.location) === null || _a === void 0 ? void 0 : _a.search)) {
```

Reply:
Good catch. It makes more sense. Thanks!